crates/ethereum/primitives/src/lib.rs

```diff
@@ -14,9 +14,10 @@ extern crate alloc;
 mod receipt;
 pub use receipt::*;
 
-/// Kept for concistency tests
+/// Kept for consistency tests
 #[cfg(test)]
 mod transaction;
```

Comment:
ah I forgot, the entire mod transaction module is only used for testing and we can phase that out entirely.

but this means we need to move the alias def here, into lib.rs